Aegis class long term care vessel

Draggeru opened the pull request:
## About The Pull Request

Adds the SSV Aegis, a roleplay-focused medical and service specialized ship to aid or take over rescue class ship cases that are above the critical care level that the rescue class level can handle, such as psychological treatment or long-term patient boarding.

## Why It's Good For The Game
Adds another syndicate-aligned faction medical/service ship(mostly roleplay oriented), this ship was designed with not only a mid/high crew in mind but with the added potential to accept and host other non-crew members for a length of time for whatever roleplay flavor. I really need to stress this ship was designed with a primary roleplay focus, it was designed in a way where people could spend weeks and even months on this one ship alone. This is what sets the Aegis apart from the Geneva, the Geneva is more gameplay oriented in comparison. The Aegis also features a joinable psychologist role and a joinable long-term patient role. Both of these roles are roleplay oriented.

![aegisfinal4](https://user-images.githubusercontent.com/81540056/179382671-5b14308f-9ebe-4dbc-8aec-98cc7252c0c5.png)

![aegisfinalingame](https://user-images.githubusercontent.com/81540056/179382676-8cc475bf-25fa-4c01-8575-0f7c04b3eb89.JPG)

For lore: Your outstanding orders from the NSV are to respond to any major critical care situations where the patient may potentially need to be onboarded for any length of time, or in cases where first response ships have a major patient overflow. While you are not necessarily a first response vessel you are able to uphold such needs in the case of an emergency where rescue class vessels cannot respond.

The Aegis does NOT have any form of research potential, only housing a singular medical tech fab. If you somehow end up with research potential you should not spec into cargo/engineering/security. Your primary focus is to uphold NSV's orders and provide the care you specialize in.

## Changelog
:cl:
add: Adds the Aegis, a ship designed primarily for medical and service(and roleplay)
:cl:

Draggeru committed:
The SAUCE.

Yellow-Mushroom commented:
I feel as if this ship strongly overlaps with the syndicate Geneva class, with these both being syndicate medical ships with an included botany.

Draggeru commented:
Geneva uses oil thrusters, contains a considerable amount of equipment for its size on top of being a major size difference, and is meant as a rescue vessel than an actual hospital in terms of roleplay flavor. Geneva also contains an emag.

Draggeru committed:
Makes the plants onboard look more dynamic and adds an external access point while I buy time to find the one thing missing a singular "

Draggeru committed:
this will fix it

Draggeru committed:
Merge branch 'shiptest-ss13:master' into Aegis

Draggeru committed:
no...THIS will fix it

Draggeru committed:
Merge branch 'Aegis' of https://github.com/Draggeru/ShiptestDragFork into Aegis

Draggeru committed:
AAAAAAAAAAAAA

Draggeru committed:
I hate atmos pipes I hate atmos pipes

Draggeru committed:
NOW its fixed

Draggeru committed:
Swapped the tanks around, lets see if that works

Draggeru committed:
With enough brute force it will work

Draggeru committed:
Adds blast doors and a button to the ships thrusters

Draggeru committed:
fixes what I broke

Draggeru committed:
Fixes the walls so they are not three tiles thick

Draggeru commented:
This is officially done and ready for maintainers to pick at, I do not have any plans to alter it at this time

Yellow-Mushroom commented:
Unsure if Im just misremembering how it looked, but with the new commits, could the PR get an updated pic?

ZephyrTFA commented:
I've updated the PR body to contain the mapdiffbot rendered image

Draggeru commented:
I have additional ideas for this PR and will be moving it to a draft after maptainers give the current version a look over. Current plans are to: reduce the chem lab to just boards that need to be built, replace the chem area with a psychologist office, shuffle the roles around to add psychologist and an "Onboard Patient" roleplay roles, and tweak the external access so it is in a more logical place. This ship is for the NSV and while it does overlap with the Geneva I think the roleplay focus, size, and the fact the NSV would have more ships of this type than most factions could justify the Aegis co exsisting with the Geneva

Edit:
Never mind, I wasn't patient enough to do this. New changes: Shifted the roles around and removed two deck assistant slots. Added one psychologist slot, one long-term patient slot, changed parts of the ship around that made the walls (specifically in cargo and in dorms) look better, and added a psychologist office, reduced all the chemistry equipment to just tech boards tucked away in medical storage.

![aegisfinalforreal](https://user-images.githubusercontent.com/81540056/178196809-772c2928-9797-4252-a1e0-4a6774576d48.png)

Draggeru committed:
Nevermind I just did the thing anyways because I have no life

Draggeru committed:
It apparently didnt like a custom area name

Draggeru committed:
More pipe shit

Fikou commented on _maps/configs/aegis.json:
```diff
@@ -0,0 +1,39 @@
+		"outfit": "/datum/outfit/job/miner/syndicate",
+		"slots": 1
+	},
+	"Ship Psycologist": {
```
misspelled

Latentish reviewed (changes requested):
If I'm very honest with you, I'm not super impressed with this ship. Part of the reason I asked you to focus on ruin making first was so that you could also learn a lot of our more obscure requirements.
- Double walls are generally something you should avoid, which are all over this ship.
![image](https://user-images.githubusercontent.com/55075747/178367042-07a0ce29-a543-4f08-83f4-f729b7f775b9.png)
- The general shape of the ship isn't bad, it has a sort of star wars rebel alliance symbol going on. But the silhouette is rather chunky. I'd like to see you smooth out your ship's outline with more diagonal and smooth walls. As is you got this jpeg edge artifacting look to it, like you used the fill tool to make the shape but didn't trim it up.
![image](https://user-images.githubusercontent.com/55075747/178367609-dbcb4957-44db-4bba-8658-9d4e2d23b629.png)
- I like that you squeezed the botany into this little area, but botany on an NSV medical ship seems a bit out of place. I'm also just not a fan of the shape of this botany in general, as it's basically just a thin long rectangle.
![image](https://user-images.githubusercontent.com/55075747/178367772-21922388-7651-4ca9-a45c-618e1012d82f.png)
- I like the medbay, I'd even suggest expanding it a bit.
![image](https://user-images.githubusercontent.com/55075747/178367870-0685dd8e-9c09-4043-95f2-880acc331aef.png)

Draggeru committed:
Adds a blood crate and gives the commanding officer a sword

Draggeru commented:
New checklist before the next review:
[ ] Ensure there are no double walls
[ ] Restore the original shape of the ship, which is more symmetrical in the original first iteration
[ ] Scrunch? Botany down more so it is not as prominent but still useable for the kitchen
[ ] Rework medical to be larger

Draggeru committed:
Remaps a bunch of stuff, fixes the typo

Draggeru committed:
Remaps a lot of the ship, fixes the typo

Draggeru committed:
Alters the map again, alters job slots

Draggeru committed:
Atmos is going to be my 13th reason why

Draggeru committed:
Completes the aegis for another review

Apogee-dev commented:
Don't forget to update the ship's image at the top of the PR.

Draggeru committed:
adds a tiiiny bit more detail

Draggeru committed:
Adds some fluff clothing for any roleplay stuff, adds a service techfab board

Draggeru commented:
This is officially done and ready for review, if I make any more changes that are not fixes yell at me

Draggeru committed:
how the fuck did I manage to break ANYTHING.

Latentish commented:
I don't have the time for a real review right now but this is a massive improvement.

Latentish commented:
![image](https://user-images.githubusercontent.com/55075747/179348611-b303f4ac-6e08-404b-a101-b10b00efb0b7.png)
This is very nice.

Latentish commented:
Last comment was me being stupid and blind. This is great. I love the maint aspect. Way to pack everything in.
![image](https://user-images.githubusercontent.com/55075747/179348765-1c862275-cd49-4cc2-a286-2a95f3e2f82e.png)

Latentish commented:
![image](https://user-images.githubusercontent.com/55075747/179348800-cc7e5d18-7b33-4f61-9d5a-94c2e6cfc107.png)
This feels empty, you should add stools in front of the bar, and add some clutter to this hallway.

Latentish reviewed (changes requested):
Looking great so far! And fuck you you're getting a review anyway.
- Add a smartfridge to botany please. You could even put it in between the bar and botany, in the wall.
![image](https://user-images.githubusercontent.com/55075747/179348917-c9c744ef-b4a0-4929-8be4-3c03155e49b3.png)
- The hallway infront of the bar seems really empty. You should add some stools in front of the bar counter, and put some clutter in the hallway. Tables, vending machines, etc.
- Optionally you could give them some more cooking equipment. An oven would be really appropriate for this setting, could put a little minikitchen area in that empty strip of floor in botany. But if you think it impedes the goal or balance of the ship that's more flourish than anything.
- I can't tell if you added one, but make sure there is a cargo/outpost communications console somewhere, with an area to receive packages.
- I can see you used the base path of "clutter" for your floors. You shouldn't use this, it's description is "this is clutter", and it's just some grey pixels. Its subtypes are what's meant to be used. "clutter" is somewhat out of place in general. (I know I said add clutter earlier but I meant like furniture just to be clear).
- Add a shower somewhere.

Draggeru commented:
> Looking great so far! And fuck you you're getting a review anyway.
> 
> * Add a smartfridge to botany please. You could even put it in between the bar and botany, in the wall.
>   ![image](https://user-images.githubusercontent.com/55075747/179348917-c9c744ef-b4a0-4929-8be4-3c03155e49b3.png)
> * The hallway infront of the bar seems really empty. You should add some stools in front of the bar counter, and put some clutter in the hallway. Tables, vending machines, etc.
> * Optionally you could give them some more cooking equipment. An oven would be really appropriate for this setting, could put a little minikitchen area in that empty strip of floor in botany. But if you think it impedes the goal or balance of the ship that's more flourish than anything.
> * I can't tell if you added one, but make sure there is a cargo/outpost communications console somewhere, with an area to receive packages.
> * I can see you used the base path of "clutter" for your floors. You shouldn't use this, it's description is "this is clutter", and it's just some grey pixels. Its subtypes are what's meant to be used. "clutter" is somewhat out of place in general. (I know I said add clutter earlier but I meant like furniture just to be clear).
> * Add a shower somewhere.

-Consider it done
-Consider it done
-By oven do you mean microwave oven? There is already a microwave on the kitchen side of the service area. If not I'm not sure what you're referring to. Additionally I can add the boards for more machines, such as a few more hydroponic tray boards and cooking machine boards. I would like to leave the space to give players a bit of freedom to alter it themselves. I think adding the boards would let them do that better anyways.
-Cargo console is on the bridge itself, I'll make sure the area is set and not broken
-Will fix
-There is already a shower in medical, you cannot see it but there is a directional window separating the bed and the shower there. Are you wanting another shower?

Draggeru committed:
Updates the ship with more things

Draggeru committed:
Does the things requested

Draggeru commented:
![aegisfinal2electricbogaloo](https://user-images.githubusercontent.com/81540056/179369827-0d9e191f-98fa-44e0-b75a-395d2f5b1f92.png)

Latentish commented:
> -Consider it done -Consider it done -By oven do you mean microwave oven? There is already a microwave on the kitchen side of the service area. If not I'm not sure what you're referring to. Additionally I can add the boards for more machines, such as a few more hydroponic tray boards and cooking machine boards. I would like to leave the space to give players a bit of freedom to alter it themselves. I think adding the boards would let them do that better anyways. -Cargo console is on the bridge itself, I'll make sure the area is set and not broken -Will fix -There is already a shower in medical, you cannot see it but there is a directional window separating the bed and the shower there. Are you wanting another shower?

- I meant the normal convection oven. But citcuits work fine for this.
-  When it comes to a shower, yeah a shower in medical is always a good addition. But from an rp and living perpective it would be kinda whack that a patient would need to go to medical just to take a shower. Try to put it near the normal bothrooms, or in there outright if you can manage.

Draggeru committed:
Fixes the walls so they dont look awful when in game

Draggeru committed:
Adds some details and widens the southern external airlock to two tiles

Draggeru committed:
It is now time to fight with atmos once again

Draggeru committed:
If I brute force atmos enough it will work

Draggeru committed:
Fuck fuck fuck

Draggeru committed:
Fixes the plasma engines

Draggeru committed:
Touch ups

Draggeru committed:
Fixes a couple things my blind ass forgot

Latentish commented:
> ![aegisfinal2electricbogaloo](https://user-images.githubusercontent.com/81540056/179369827-0d9e191f-98fa-44e0-b75a-395d2f5b1f92.png)

Good.

Latentish reviewed (approved):
Seems pretty sanitary so far. I'd like to see it get tested. TM?

Draggeru commented:
After discussing issues with my repository with maintainers this map pr is moved to #1391 on a new repository.
